tasks.robot

```diff
@@ -1,49 +1,73 @@
 *** Settings ***
-Documentation       Template robot main suite.
+Documentation
+...                 This robot uses the provided work item to select a site secret
+...                 and then log into that site and set a new password. The website
+...                 implementation is not complete and should be done by the user of
+...                 this template.
 
 Library             DateTime
 Library             FakerLibrary
 Library             RPA.Robocorp.Vault
 Library             Collections
+Library             RPA.Robocorp.WorkItems
 
 
 *** Tasks ***
 Minimal task
-    ${currentPass}=    Get Current Password
-    ${newPass}=    Create password
-    #Change my password
-    Store my password    ${currentPass}    ${newPass}
+    ${current_pass}=    Get current password
+    ${new_pass}=    Create password
+    Change my password    ${current_pass}    ${new_pass}
+    Store my password    ${current_pass}    ${new_pass}
     Log    Done.
 
 
 *** Keywords ***
-Get Current Password
-    [Documentation]    connect to Control Room Vault and loads the myURL secret
-    ...    items to change for your environment
-    ...    myURL = change to the secret name stored in your Vault
+Get current password
+    [Documentation]
+    ...    Using the work item variable ``site_secret``, this keyword pulls down the
+    ...    associated secret from the Control Room Vault. Provide the name of the vault
+    ...    item in that work item variable. The Vault item should have the password
+    ...    stored in the value ``password``.
 
-    ${secret}=    Get Secret    myURL
+    ${site_secret}=    Get work item variable    site_secret
+    ${secret}=    Get Secret    ${site_secret}
     RETURN    ${secret}
 
 Create password
-    [Documentation]    uses FakerLibrary to generate a new random password
-    ...    if you do not need special characters, upper, lower, etc set those to False
+    [Documentation]
+    ...    Uses FakerLibrary to generate a new random password of the provided
+    ...    ``length`` (defaults to 10 characters long). if you do not need special
+    ...    characters, upper, lower, etc set those to False
+    [Arguments]    ${length}=10    ${special_chars}=${True}    ${digits}=${True}
```

Comment:
can you switch it to use work items instead?

Reply:
Okay, new commit I just pushed fixes this.